package.json

```diff
@@ -240,7 +240,7 @@
     "ts-dedent": "^1.1.1",
     "ts-jest": "^26.1.0",
     "ts-node": "^8.9.1",
-    "typescript": "^3.4.0",
+    "typescript": "^3.7.0",
```

Comment:
Typescript 3.9 is already pinned in the `yarn.lock`, but since I used optional chaining which requires at least typescript 3.7, I figured this should be changed

Reply:
I think you can set `^3.9.6` directly because as you said we are already using it

Reply:
Fair enough!

@gaetanmaisse it looks like there are a lot of duplicates in `yarn.lock` (like, `typescript` shows up 15 times in `yarn why typescript`), it might be a good idea to run `yarn-deduplicate` to clean that up.

Reply:
True! I plan to do a big dep clean and dedup work in Storybook 6.1 :)